### PR TITLE
Initial Twine JSON parsing

### PR DESCRIPTION
**Pull request by grahamearley**
- Added `SimpleJson` plugin for parsing JSON in C#
- Added a script for reading data from each node in the story (read from JSON)
- Modified Charlie’s “Assets > Import Twine Data…” GUI so that when you click import, it calls the JSON reader and (for now) prints out some data from the story
#### TODO:
- Make the file importer take a JSON file and read _that_ file (for now, it’s ignoring whatever file you choose and just using a hardcoded json string)
- Use the twine data!!

### CODE REVIEW

File: Project/Assets/Editor/Twine Importer/TwineJsonParser.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+		// Demo code, printing the ID and name of each node:
+		foreach (JSONNode storyNode in parsedArray) {
+			Debug.Log ("Node ID: " + storyNode[NodeIdKey]);
+			Debug.Log ("Node name: " + storyNode[NodeNameKey]);
+		}
+
+	}
+
+
+}
```
Comment (grahamearley) [inline Project/Assets/Editor/Twine Importer/TwineJsonParser.cs:25]: Note: this is the last of the code I wrote. The files below are from the SimpleJson package

**Review by cpimhoff — CHANGES_REQUESTED** [review]
Your code looks great, and this seems like a fine start to grabbing the Twine data from JSON.

If hesistent to pull these in before we move around the content currently located at `Assets/JSON Tests`. If this is simply a suite of tests for the JSON parser we're utilizing then we don't need it, and it shouldn't be included.

It appears to me we do not require the contents of `JSON Tests`, just what's located at `Assets/plugins/SimpleJSON.cs` (and some meta files associated with it).

Especially when including a dependency, let's commit the absolute minimum.

**Comment by grahamearley** [comment]
@cpimhoff Good point. I removed the test files.

**Comment by brennan724** [comment]
I like it.

**Pull request merged**